pages/common/ember.md

```diff
@@ -0,0 +1,32 @@
+# ember
+
+> The Ember.js command line utility.
+> Used for creating and maintaining Ember.js applications.
+
+- Create a new Ember application:
+
+`ember new {{my_new_app}}`
+
+- Create a new Ember addon:
+
+`ember addon {{my_new_addon}}`
+
+- Build the project:
+
+`ember build`
+
+- Run the development server:
+
+`ember serve`
+
+- Run the test suite:
+
+`ember test`
+
+- Generate files:
```

Comment:
Can you expand this a bit further ? It's not exactly clear what you mean by generate files.

Reply:
For sure. I actually just grabbed the headings and examples from the Ember CLI readme, but I can definitely expand it more.